src/client/App.js

```diff
@@ -30,6 +41,36 @@ function App() {
         <SignIn exact path="/sign-in" />
         <SignUp exact path="/sign-up" />
         <ResetPassword exact path="/reset-password" />
+        <Route exact path="/error">
```

Comment:
The 404 page will be triggered just if the user types the word error on the path, so you probably want to approach this in a different way. I am sure you will be able to find a solution in google :)